warp-core-gatling/src/test/scala/com/workday/warp/junit/GatlingTestSpec.scala

```diff
@@ -0,0 +1,26 @@
+package com.workday.warp.junit
+
+import java.util.concurrent.TimeUnit
+
+import com.workday.warp.adapters.gatling.{GatlingJUnitRunner, WarpSimulation}
+import io.gatling.core.Predef._
+import io.gatling.core.structure.{PopulationBuilder, ScenarioBuilder}
+import io.gatling.http.Predef._
+import org.junit.runner.RunWith
+
+import scala.concurrent.duration.FiniteDuration
+
+/**
+  * Created by tomas.mccandless on 7/1/20.
+  */
+@RunWith(classOf[GatlingJUnitRunner])
+class GatlingTestSpec extends WarpSimulation {
+
+  val scn: ScenarioBuilder = scenario("Google test")
+    .exec(http("Google")
+      .get("https://www.google.comjdah")
+      .check(status is 100))
```

Comment:
not too familiar with the `100` http status code here, do we not get an unable to resolve host error?

Reply:
we do get an unresolvable host error, but it doesn't fail the build, curiously.

i just wanted us to be aware of that, i'll add some more comments